packages/material-ui-lab/src/MonthPicker/MonthPicker.tsx

```diff
@@ -1,16 +1,24 @@
 import * as React from 'react';
 import { createStyles, WithStyles, withStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
 import PickersMonth from './PickersMonth';
 import { useUtils, useNow } from '../internal/pickers/hooks/useUtils';
 import { PickerOnChangeFn } from '../internal/pickers/hooks/useViews';
 
 export interface MonthPickerProps<TDate> {
   date: TDate | null;
+  /** Minimal selectable date */
   minDate: TDate;
+  /** Maximal selectable date */
   maxDate: TDate;
+  /** Callback fired on date change */
   onChange: PickerOnChangeFn<TDate>;
+  /** If `true` past days will be disabled */
```

Comment:
present tense, comma, leading dot, multi-line block comment:

```suggestion
  /**
   * If `true`, past days are disabled.
   * /
```